tf-modules/archive/replay_sqs_messages.tf

```diff
@@ -0,0 +1,95 @@
+data "aws_iam_policy_document" "replay_sqs_messages_policy" {
+
+  statement {
+    actions   = [
+      "s3:GetObject",
+      "s3:PutObject",
+      "s3:DeleteObject"
+    ]
+    resources = ["arn:aws:s3:::${var.system_bucket}/*"]
+  }
+
+  statement {
+    actions = [
+      "ec2:CreateNetworkInterface",
+      "ec2:DescribeNetworkInterfaces",
+      "ec2:DeleteNetworkInterface"
+    ]
+    resources = ["*"]
+  }
+
+  statement {
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:DescribeLogStreams",
+      "logs:PutLogEvents"
+    ]
+    resources = ["*"]
+  }
+  statement {
+    actions = [
+      "secretsmanager:GetSecretValue"
+    ]
+    resources = [var.rds_user_access_secret_arn]
+  }
+
+    statement {
+      actions = [
+        "sqs:SendMessage",
+        "sqs:GetQueueUrl",
+      ]
+      resources = ["arn:aws:sqs:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:*"]
+  }
+}
+
+resource "aws_iam_role" "replay_sqs_messages_role" {
+  name                 = "${var.prefix}_replay_sqs_messages_role"
+  assume_role_policy   = data.aws_iam_policy_document.lambda_assume_role_policy.json
+  permissions_boundary = var.permissions_boundary_arn
+  tags                 = var.tags
+}
+
+resource "aws_iam_role_policy" "replay_sqs_messages_role_policy" {
+  name   = "${var.prefix}_replay_sqs_messages_lambda_role_policy"
+  role   = aws_iam_role.replay_sqs_messages_role.id
+  policy = data.aws_iam_policy_document.replay_sqs_messages_policy.json
+}
+
+resource "aws_lambda_function" "replay_sqs_messages" {
+  filename         = "${path.module}/../../packages/api/dist/replaySqsMessages/lambda.zip"
+  source_code_hash = filebase64sha256("${path.module}/../../packages/api/dist/replaySqsMessages/lambda.zip")
+  function_name    = "${var.prefix}-replaySqsMessages"
+  role             = aws_iam_role.replay_sqs_messages_role.arn
+  handler          = "index.handler"
+  runtime          = "nodejs12.x"
+  timeout          = 300
+  memory_size      = 512
+
+  environment {
+    variables = {
+      system_bucket                = var.system_bucket
+      stackName                    = var.prefix
+      databaseCredentialSecretArn  = var.rds_user_access_secret_arn
```

Comment:
Does this lambda talk to RDS directly? If not, then this environment variable isn't necessary

Reply:
[`030ed97` (#2373)](https://github.com/nasa/cumulus/pull/2373/commits/030ed97f35a8892454a2f46e229a87b2bb447203#diff-7b93858158e237529239ab394d5f5c6428fd1d94d04af74213f3bad9f3606ea5L73)